transform/mattermost-analytics/models/intermediate/product/configs/int__config_ldap.sql

```diff
@@ -0,0 +1,101 @@
+{{
+    config({
```

Comment:
These will be removed before checkin.

Reply:
cc: @ifoukarakis , actually I take that back. Because of the errors here: https://cloud.getdbt.com/deploy/13604/projects/216476/runs/340761541/ , I don't think I can remove the materialized statement.